src/views/bootablevolumes/actions/components/DeleteBootableVolumesModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+import { Trans } from 'react-i18next';
+
+import { DEFAULT_PREFERENCE_LABEL } from '@catalog/CreateFromInstanceTypes/utils/constants';
+import DataSourceModel from '@kubevirt-ui/kubevirt-api/console/models/DataSourceModel';
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import DeleteModal from '@kubevirt-utils/components/DeleteModal/DeleteModal';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useLastNamespacePath } from '@kubevirt-utils/hooks/useLastNamespacePath';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+
+type DeleteBootableVolumesModalProps = {
+  dataSource: V1beta1DataSource;
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const DeleteBootableVolumesModal: FC<DeleteBootableVolumesModalProps> = ({
+  dataSource,
+  isOpen,
+  onClose,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const lastNamespacePath = useLastNamespacePath();
+
+  const labelsWithoutDefaultPreference = Object.keys(dataSource?.metadata?.labels)
```

Comment:
can be exported to util file?

Reply:
Of course, it can. I just don't see any reason to do that. It's not gonna be used anywhere outside this modal component. And if yes, we always can change that later.